inference-engine/src/gna_plugin/gna_plugin.cpp

```diff
@@ -18,7 +18,6 @@
 #include <utility>
 #include <limits>
 
-#include <low_precision_transformations/blob_transformation.hpp>
```

Comment:
could you please also remove `inference_engine_lp_transformations` dependency for plugins in cmake?

Reply:
done.